README.md

```diff
@@ -541,3 +541,58 @@ for sparsity patterns. In the future, those two packages should just depend on
 ArrayInterface.jl and remove this issue entirely from the user space.
 
 Additionally, GPUs need ArrayInterfaceGPUArrays for proper determination of the indexing.
+
+## Reproducibility
+```@raw html
+<details><summary>The documentation of this SciML package was built using these direct dependencies,</summary>
```

Comment:
wait, this is the README, not the docs.

Reply:
Ohh, yeah. Sorry. You are right. I need to put this into the index.md not the README. I just looked at the text and not the file name. Let me fix that then.

Reply:
Okay, this should be fixed now. I moved that text from the README to the index.md. Sorry again.